shared_model/interfaces/commands/impl/remove_peer.cpp

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "interfaces/commands/remove_peer.hpp"
+
+#include "cryptography/public_key.hpp"
+
+namespace shared_model {
+  namespace interface {
+
+    std::string RemovePeer::toString() const {
+      return detail::PrettyStringBuilder()
+          .init("RemovePeer")
+          .append(pubkey().toString())
```

Comment:
```suggestion
          .append("public key", pubkey().toString())
```

Reply:
https://github.com/hyperledger/iroha/blob/4e0cfa3a2188e1028b018e3fe9fbfe26fb26caea/shared_model/cryptography/model_impl/public_key.cpp#L19
So it should not be required to add another string.